src/chapters/radius-auth.xml

```diff
@@ -0,0 +1,251 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<chapter xml:id="radius-auth" xmlns="http://docbook.org/ns/docbook" version="5.0" xml:lang="en"
+    xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <title>RADIUS Authentication</title>
+    <indexterm>
+        <primary>RADIUS Authentication</primary>
+    </indexterm>
+    <para>Guacamole supports delegating authentication to a RADIUS service, such as FreeRADIUS, to
+        validate username and password combinations, and to support multi-factor authentication.  This
+        authentication method must be layered on top of some other authentication extension, such as
+        those available from the main project website, in order to provide access to actual
+        connections.</para>
+    <section xml:id="radius-downloading">
+        <title>Downloading the RADIUS authentication extension</title>
+        <para>The RADIUS extension depends on software that is covered by a LGPL license, which is
+            incompatible with the Apache 2.0 license under which Guacamole is licensed.  Due to this
+            dependency, the Guacamole project cannot distribute binary versions of the RADIUS extension.
+            If you want to use this extension you will need to build the code - or at least the RADIUS
+            extension yourself.  Build instructions can be found in the section
+            <xref linkend="installing-guacamole"/>.</para>
+    </section>
+    <section xml:id="installing-radius-auth">
+        <title>Installing RADIUS authentication</title>
+        <para>The RADIUS extension must be explicitly enabled during build time in order to generate
+            the binaries and resulting JAR file.  This is done by adding the flag <option>-Plgpl-extensions</option>
+            to the MAVEN command line during the build, and should result in the output below:</para>
```

Comment:
Maven is not an acronym (see http://maven.apache.org/).

Reply:
Fixed.